docs/cloud/managing-airbyte-cloud.md

```diff
@@ -84,6 +74,37 @@ You can use one or multiple workspaces with Airbyte Cloud.
 | Single               | You can use the same payment method for all purchases.                        | Credits pay for the use of resources in a workspace when you run a sync. Resource usage cannot be divided and paid for separately (for example, you cannot bill different departments in your organization for the usage of some credits in one workspace).                                     |
 | Multiple             | Workspaces are independent of each other, so you can use a different payment method card for each workspace (for example,  different credit cards per department in your organization). | You can use the same payment method for different workspaces, but each workspace is billed separately. Managing billing for each workspace can become complicated if you have many workspaces. |
 
+### Switch between multiple workspaces
+
+To switch between workspaces:
+
+1. On the [Airbyte Cloud](http://cloud.airbyte.io) dashboard, click the current workspace name under the Airbyte logo in the navigation bar.
+
+2. Click **View all workspaces**.
+
+3. Click the name of the workspace you want to switch to.
+
+### Choose your default data residency
+Default data residency allows you to choose where your data is processed. When you set the default data residency, it applies that data residency to all new connections, but it will not affect existing connections. 
+
+For individual connections, you can choose a data residency that is different from the default. You can do this in the [connection settings](#choose-the-data-residency-for-a-connection) or when you create a [new connection](https://docs.airbyte.com/cloud/getting-started-with-airbyte-cloud#set-up-a-connection). Your data will not leave the chosen data residency for your connection.
```

Comment:
I think this is an area where we might want to talk about the control plane vs. the data plane. Because this setting doesn't necessarily mean where your data is stored, but it's about where your data is processed.

Reply:
that's a good point - we don't highlight the difference between the control/data plane today i.e. while data is processed in the specific region, configuration information is still stored in the US.

this means that users need to be careful to make sure they don't choose a sensitive cursor.
 
Andy, do you think this is worth calling out? Or is it too 'scary' to do so?

Reply:
I think it's worth mentioning!

Reply:
I agree that the fact that data is processed in one region and stored in another needs to be mentioned. I'm wondering if I should delete the last line "Your data will not leave the chosen data residency for your connection" and add a note like "Your data is processed in the chosen residency, but some data is still stored in the US." We could also list out types of data (like configuration information) we store in the US if that would help the users.

Any thoughts on that?

Reply:
>  should delete the last line "Your data will not leave the chosen data residency for your connection" and add a note like "Your data is processed in the chosen residency, but some data is still stored in the US." 

I like that. I would like the note to be a markdown note so it jumps out.

One tactical point, we don't have to wordsmith too much in this PR. We can merge this in for now and iterate on subsequent PRs.

Reply:
@andyjih @davinchia Do you think that a note like this would have enough info about data plane and control plane? Is there any other info you think the users will need to know?  (I'll edit wording after we know what info we want to include):

Your data is processed on a data plane in the chosen residency, but some data, like data associated with the sync mode, cursor, and primary key, is still stored in the US on our control plane.

Reply:
That's pretty good. I would also add: "Please use caution when selecting the cursor and primary key columns."